x/dualstaking/keeper/delegator_reward.go

```diff
@@ -110,6 +118,14 @@ func (k Keeper) CalcEffectiveDelegationsAndStake(stakeEntry epochstoragetypes.St
 // CalcDelegatorReward calculates a single delegator reward according to its delegation
 // delegatorReward = delegatorsReward * (delegatorStake / totalDelegations) = (delegatorsReward * delegatorStake) / totalDelegations
 func (k Keeper) CalcDelegatorReward(delegatorsReward math.Int, totalDelegations math.Int, delegation types.Delegation) math.Int {
+	// Sanity check - totalDelegations > 0
+	if totalDelegations.IsZero() {
+		utils.LavaFormatWarning("critical: Attempt to divide by zero", nil,
```

Comment:
same comment as last one

Reply:
Fixed in 20bf6f5ad2b263ea30b7641558c140ceca7a9293